lib/meta_logger.ex

```diff
@@ -9,6 +9,8 @@ defmodule MetaLogger do
   @type metadata :: keyword()
   @metadata ~w(dictionary $logger_metadata$)a
 
+  defdelegate format(payload), to: MetaLogger.Formatter
```

Comment:
You can add docs and specs to this new function.

Reply:
I'm thinking now, do we need this? Or, if needed, it can be accessed from `MetaLogger.Formatter` directly?